spec/models/profile_spec.rb

```diff
@@ -17,6 +17,10 @@
   end
 
   describe "Validations" do
+    it 'is valid with valid attributes' do
+      profile = (build(:profile))
+      expect(profile).to be_valid
```

Comment:
does this work? Should there always be a user for this to be valid, or am I misunderstanding? Or another way of putting it, does build profile create an empty profile with no user?

Reply:
nevermind, didn't know how factory worked